tests/MenderAPI/authentication.py

```diff
@@ -101,12 +101,15 @@ def get_auth_token(self, create_new_user=True):
                     self.create_user(self.username, self.password)
 
             # It might take some time for create_org to propagate the new user.
-            # Retry login for a minute.
-            for i in range(60):
+            # Retry login for some time.
+            try_number=0
+            for i in range(256):
+                time.sleep(8)
```

Comment:
Don't you think that 34 minutes is an overkill? I know you love these numbers that I don't see they make much sense... I would say that if I can't log in in couple of minutes is enough to abort and fail the test.

Reply:
My point is that retrying so much just for the sake of having a green test might hide other robustness or performance issues. From the QA point of view: we are not scared about failures, we test to **find** problems before our users do ;)